src/main/java/com/ericsson/ei/waitlist/WaitListStorageHandler.java

```diff
@@ -44,18 +44,21 @@
 @Component
 public class WaitListStorageHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(WaitListStorageHandler.class);
+    public static final String DOCUMENT_ID_KEY = "_id";
+    public static final String DOCUMENT_TIME_KEY = "Time";
```

Comment:
Was there any idea to put these constants in one file and use them from there instead of redeclaring them in each file?

Reply:
Indeed, a better idea. I'll move them out to a constant class in the Mongo package (since these in particular is only related to Mongo documents)